openai-assistant/META-INF/MANIFEST.MF

```diff
@@ -14,6 +14,7 @@ Require-Bundle: org.eclipse.core.runtime,
  ch.ivyteam.ivy.webservice.execution;resolution:=optional,
  ch.ivyteam.lib.jackson,
  ch.ivyteam.ivy.jersey.client,
- javax.ws.rs
+ javax.ws.rs,
+ ch.ivyteam.ivy.rest.client.exec;resolution:=optional
```

Comment:
this bundle is new, was previously combined into another one. 
so we add it as 'optional' dependency ... in order that lts10 build can also comply with it.